test/simulation/xudtest/node.go

```diff
@@ -50,7 +51,6 @@ func (cfg nodeConfig) genArgs() []string {
 	var args []string
 
 	args = append(args, "--initdb=false")
-	args = append(args, "--nosanitychecks=true")
```

Comment:
It's great that we get to test sanity checks and swaps now with simulation tests.